pkg/asset/installconfig/powervs/client.go

```diff
@@ -587,28 +587,28 @@ func (c *Client) GetVPCByName(ctx context.Context, vpcName string) (*vpcv1.VPC,
 	if err != nil {
 		return nil, fmt.Errorf("failed to list vpc regions: %w", err)
 	}
-
+	var vpcNamesList []string
 	for _, region := range listRegionsResponse.Regions {
 		err := c.vpcAPI.SetServiceURL(fmt.Sprintf("%s/v1", *region.Endpoint))
 		if err != nil {
 			return nil, fmt.Errorf("failed to set vpc api service url: %w", err)
 		}
-
 		vpcs, detailedResponse, err := c.vpcAPI.ListVpcsWithContext(ctx, c.vpcAPI.NewListVpcsOptions())
 		if err != nil {
 			if detailedResponse.GetStatusCode() != http.StatusNotFound {
 				return nil, err
 			}
 		} else {
 			for _, vpc := range vpcs.Vpcs {
+				vpcNamesList = append(vpcNamesList, *vpc.Name)
 				if *vpc.Name == vpcName {
 					return &vpc, nil
 				}
 			}
 		}
 	}
 
-	return nil, errors.New("failed to find VPC")
+	return nil, errors.New(fmt.Sprint("failed to find VPC. VPC name entered in platform.powervs.vpcName ", vpcName, " is not among ", vpcNamesList))
```

Comment:
```suggestion
	return nil, fmt.Errorf("failed to find VPC %q. Available VPCs: %v", vpcName, vpcNamesList)
```

nit: We use `fmt.Errorf` here for consistency?

I suggested to also not mention the installconfig field (i.e. `platform.powervs.vpcName`) here. That should be handled outside of `GetVPCByName`, wherever it is called.